src/display/canvas.js

```diff
@@ -478,6 +472,7 @@ var CanvasGraphics = (function CanvasGraphicsClosure() {
     this.smaskCounter = 0;
     this.tempSMask = null;
     this.cachedCanvases = new CachedCanvases();
+    this.DOMcanvasFactory = new DOMcanvasFactory();
```

Comment:
It may be better to call this `this.canvasFactory`, since I believe that one of the points of issue #7798 is to allow an API consumer to provide their own canvas factory (e.g. using Node canvas), and then a using the `this.DOMCanvasFactory` name here could be confusing.